pkg/api/unversioned/group_version.go

```diff
@@ -29,8 +29,15 @@ type GroupResource struct {
 	Resource string
 }
 
+func (gr GroupResource) IsEmpty() bool {
+	return len(gr.Group) == 0 && len(gr.Resource) == 0
+}
+
 func (gr *GroupResource) String() string {
-	return strings.Join([]string{gr.Group, ", Resource=", gr.Resource}, "")
+	if len(gr.Group) == 0 {
+		return gr.Resource
+	}
+	return gr.Resource + "." + gr.Group
```

Comment:
this doesn't bother me for display, but I really don't want someone trying to parse this... not sure how to prevent that